pkg/networkmanager/openvpn.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useContext, useState } from 'react';
+import { Name, NetworkModal, dialogSave } from "./dialogs-common";
+import { FileUpload } from '@patternfly/react-core/dist/esm/components/FileUpload/index.js';
+import { FormGroup } from '@patternfly/react-core/dist/esm/components/Form/index.js';
+import { TextInput } from '@patternfly/react-core/dist/esm/components/TextInput/index.js';
+import cockpit from 'cockpit';
+import { ModelContext } from './model-context';
+import { useDialogs } from 'dialogs.jsx';
+
+const _ = cockpit.gettext;
+
+export function OpenVPNDialog({ settings, connection, dev }) {
+    const Dialogs = useDialogs();
+    const idPrefix = "network-openvpn-settings";
+    const model = useContext(ModelContext);
+
+    const [iface, setIface] = useState(settings.connection.interface_name);
+    const [remote, setRemote] = useState("");
+    const [caCertName, setCaCertName] = useState("");
+    const [caCertVal, setCaCertVal] = useState("");
+    const [userCertName, setUserCertName] = useState("");
+    const [userCertVal, setUserCertVal] = useState("");
+    const [userKeyName, setUserKeyName] = useState("");
+    const [userPrivateKey, setUserPrivateKey] = useState("");
+    const [dialogError, setDialogError] = useState("");
```

Comment:
These 9 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19363-20230921-075022-79cb9fd2-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)